linux_os/guide/system/bootloader-grub2/grub2_no_removeable_media/rule.yml

```diff
@@ -2,7 +2,7 @@ documentation_complete: true
 
 prodtype: wrlinux1019,rhel7,rhel8,ol7,ol8,rhv4
 
-title: 'Boat Loader Is Not Installed On Removeable Media'
+title: 'Boot Loader Is Not Installed On Removeable Media'
```

Comment:
should be removable

Reply:
ACtually maybe not necessarily:
https://www.collinsdictionary.com/dictionary/english/removeable